task.js

```diff
@@ -28,7 +28,13 @@ class Task {
     this.mapper = mapper || function() { return ''; };
     this.translations = [];
   }
-
+  
+  async logError(fieldName, fieldValue, recordId){
+    const otfl = `${OUTPUT_DIRECTORY}/${this.title}-errors.txt`;
```

Comment:
Use full variable names instead of abbreviations (e.g. `outputFile` instead of `otfl`. Makes it more clear for people that need to understand the code later on.